extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,38 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+let blbSlot = {
+	config: {
+		targeting: {}
+	},
+	getSlotName: () => 'BOTTOM_LEADERBOARD',
+	getId: () => 'BOTTOM_LEADERBOARD'
+};
+
+Context.extend(config);
+TemplateService.register(BigFancyAdBelow);
+SlotService.add(blbSlot);
+
+function setUapId(uapId) {
+	UniversalAdPackage.setUapId(uapId);
+}
+
+function loadCustomAd(callback) {
+	return (params) => {
+		try {
+			const slot = SlotService.getBySlotName(params.slotName);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this ES6 variable declaration with var. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AConstLet)

[project-id]: wikia:app